### PR TITLE
Suppress compliance issues for expect/actual code

### PR DESCRIPTION
**Pull request by csilvestrini**
* Moves Addressable.kt into an addressable folder to match its package
* Suppresses some Kotlin lints around filenames and package names
* Fixes some packages
* Removes some confusing aliases

### CODE REVIEW

File: src/tools/schema2kotlin.ts [hunk 1]
```diff
@@ -43,6 +43,7 @@ export class Schema2Kotlin extends Schema2Base {
   fileHeader(outName: string): string {
     const withCustomPackage = (populate: string) => this.scope !== 'arcs' ? populate : '';
     return `\
+@file:Suppress("PackageName", "TopLevelName")
```
Comment (Cypher1) [inline src/tools/schema2kotlin.ts]: Not familiar with this Kotlin warning so my review doesn't cover its use.

File: src_kt/javatests/arcs/stringEncoder/BUILD [hunk 2]
```diff
@@ -11,7 +11,7 @@ arcs_kt_jvm_test_suite(
     name = "stringEncoder",
     package = "arcs.stringEncoder",
     deps = [
-        "//src/wasm/kotlin:arcs",
+        "//src/wasm/kotlin/java/arcs",
```
Comment (Cypher1) [inline src_kt/javatests/arcs/stringEncoder/BUILD]: This seems surprising to me i.e. that we have a java dir inside our kotlin dir.

Would be interested in the meaning of this, but am not against it, just don't understand.

Comment (csilvestrini) [reply src_kt/javatests/arcs/stringEncoder/BUILD]: Yeah it's not ideal. Just want to get the copybara import to work

**Comment by csilvestrini** [comment]
> LGTM for the kt and build files. Unsure about the java dir and warning supression.

Thanks. The warning suppression is unfortunately required in order to use the Kotlin expect/actual keywords. To briefly summarise, we have `expect Foo` in `./Foo.kt` and `actual Foo` in `./wasm/Foo.kt`. The linter complains that the package in `./wasm/Foo.kt` does not match the folder. We can move `./wasm/Foo.kt` to `./WasmFoo.kt` or to `./Foo.wasm.kt` so that the package path is correct, but then the linter complains that the class should be called `WasmFoo` (or `Foo.wasm` O.o), which we can't do (must make the `expect Foo` in both class name and package)

**Pull request merged**